### PR TITLE
MAUI - 903144 - Added UG content for SelectedValue support in SfCombobox and SfAutocomplete controls .NET MAUI

### PR DESCRIPTION
**Pull request by PriyadharshniM**
**Description**
        Added UG content for SelectedValue support in SfCombobox and SfAutocomplete controls .NET MAUI

### CODE REVIEW

File: MAUI/Autocomplete/Selection.md [hunk 1]
```diff
@@ -245,32 +245,32 @@ The following gif image illustrates the result of the above code:
 
 N> SelectionChanged event arguments `CurrentSelection` and `PreviousSelection` marked as "Obsolete". You can use the `AddedItems` and `RemovedItems` event arguments.
 
-## Get the selected value
+## Getting and setting the selected value
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Selection.md:248]: Change the title.

File: MAUI/Autocomplete/Images/Selection/net-maui-autocomplete-multiple-selected-value.png
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Images/Selection/net-maui-autocomplete-multiple-selected-value.png:1]: The token is clipping. update the image @PriyadharshniM.

File: MAUI/Autocomplete/Selection.md (hunk 1 above)
Comment (PriyadharshniM) [reply MAUI/Autocomplete/Selection.md]: Title changed

File: MAUI/Autocomplete/Images/Selection/net-maui-autocomplete-multiple-selected-value.png
Comment (PriyadharshniM) [reply MAUI/Autocomplete/Images/Selection/net-maui-autocomplete-multiple-selected-value.png:1]: Image updated

File: MAUI/Autocomplete/Selection.md [hunk 2]
```diff
@@ -245,32 +245,32 @@ The following gif image illustrates the result of the above code:
...
...fusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValue) property in a Autocomplete control enables you to get or set the [SelectedValue](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValue) based on the [SelectedItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedItem) or [SelectedItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedItems) depending on the selection mode. The [SelectedValuePath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValuePath) property defines, which property of the selected item is used to populate the [SelectedValue](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValue).
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/Selection.md:250]: The [SelectedValue](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValue) property in an Autocomplete control allows you to get or set the selected value based on the [SelectedItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedItem) or [SelectedItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedItems), depending on the selection mode. The [SelectedValuePath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValuePath) property specifies which property of the selected item is used to populate the [SelectedValue](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValue).

File: MAUI/Autocomplete/Selection.md [hunk 3]
```diff
@@ -245,32 +245,32 @@ The following gif image illustrates the result of the above code:
...
 
+In single selection mode, the [SelectedValue](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValue) property contains the value corresponding to the [SelectedValuePath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValuePath)("ID"). If the [SelectedItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedItem) returns the entire object(`SocialMedia`), then [SelectedValue](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValue) holds the value of `SocialMedia.ID`.
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/Selection.md:252]: In single selection mode, the [SelectedValue](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValue) property holds the value defined by the [SelectedValuePath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValuePath) property, such as "ID". When the [SelectedItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedItem) returns the entire object (e.g., SocialMedia), the [SelectedValue](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_SelectedValue) contains the value of the SocialMedia.ID field.

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**